src/util/BaseLoginController.js

```diff
@@ -55,6 +55,7 @@ define(['okta', 'q'], function (Okta, Q) {
       }, this);
       var transactionErrorHandler = _.bind(function (err) {
         this.options.appState.set('transactionError', err);
+        this.trigger('transactionError', err);
```

Comment:
Turns out `transactionError` will eventually invoke `callGlobalError` at certain cases.
In additions, `callGlobalError` is invoked directly when error happened at some case (search `callGlobalError` through code base)
So I'm wondering we introduces some confusion to developers because there are two error handlers basically. (globalError handler and afterError handler)

Reply:
Yep - this is a known problem (or feature), with these types of events. I think it makes sense to let the current execution flow continue (callback to error handler, throw error, etc), **and** emit the event. Moving forward with our blocking events, I'd expect us to no longer carry out the `globalError` behavior.